### PR TITLE
Register Subdomain: otaviozanon

### PR DESCRIPTION
**Pull request by otaviozanon**
Added the domain: otaviozanon

### CODE REVIEW

File: domains/otaviozanon.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "otaviozanon",
+        "email": "otaviozanonn@icloud.com"
+      },
+      "record": {
+        "URL": "test"
```
Comment (WilliamDavidHarrison) [inline domains/otaviozanon.json:8]: Invalid URL

File: domains/otavio.zanon.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "otaviozanon",
+        "email": "otaviozanonn@icloud.com"
+      },
+      "record": {
+        "URL": "http://google.com"
```
Comment (WilliamDavidHarrison) [inline domains/otavio.zanon.json:8]: Please use a different URL.

File: domains/otavio.zanon.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (WilliamDavidHarrison) [inline domains/otavio.zanon.json:1]: Do you own zanon.is-a.dev?

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by WilliamDavidHarrison** [comment]
No changes due to our requirements.

If you want an is-a.dev domain please make a new PR meeting our requirements.

**Pull request closed without merging**